example_4x_adj_vbar_pak.yaml

```diff
@@ -0,0 +1,504 @@
+  channel: 30
+  universe: dmx
+  id: vbar4_effect_level
+- platform: dmx512
+  channel: 31
+  universe: dmx
+  id: vbar4_effect_select
+- platform: dmx512
+  channel: 32
+  universe: dmx
+  id: vbar4_effect_mode
+
+
+number:
+- platform: template
+  name: "VBar1 Effect Level"
+  icon: mdi:flower-pollen-outline
+  id: vbar1_effect_level_num
+  optimistic: true
+  min_value: 0
+  max_value: 255
+  step: 1
+#  mode: box
+  restore_value: true
+  set_action:
+    - lambda: |-
+        id(vbar1_effect_level).set_level(x / 255);
+- platform: template
+  name: "VBar1 Effect Mode"
+  icon: mdi:dots-hexagon
+  id: vbar1_effect_mode_num
+  optimistic: true
+  min_value: 0
+  max_value: 255
+  step: 1
+  mode: box
+  restore_value: true
+  set_action:
+    - lambda: |-
+        id(vbar1_effect_mode).set_level(x / 255);
+
+- platform: template
+  name: "VBar2 Effect Level"
+  icon: mdi:flower-pollen-outline
+  id: vbar2_effect_level_num
+  optimistic: true
+  min_value: 0
+  max_value: 255
+  step: 1
+#  mode: box
+  restore_value: true
+  set_action:
+    - lambda: |-
+        id(vbar2_effect_level).set_level(x / 255);
+- platform: template
+  name: "VBar2 Effect Mode"
+  icon: mdi:dots-hexagon
+  id: vbar2_effect_mode_num
+  optimistic: true
+  min_value: 0
+  max_value: 255
+  step: 1
+  mode: box
+  restore_value: true
+  set_action:
+    - lambda: |-
+        id(vbar2_effect_mode).set_level(x / 255);
+
+- platform: template
+  name: "VBar3 Effect Level"
+  icon: mdi:flower-pollen-outline
+  id: vbar3_effect_level_num
+  optimistic: true
+  min_value: 0
+  max_value: 255
+  step: 1
+#  mode: box
+  restore_value: true
+  set_action:
+    - lambda: |-
+        id(vbar3_effect_level).set_level(x / 255);
+- platform: template
+  name: "VBar3 Effect Mode"
+  icon: mdi:dots-hexagon
+  id: vbar3_effect_mode_num
+  optimistic: true
+  min_value: 0
+  max_value: 255
+  step: 1
+  mode: box
+  restore_value: true
+  set_action:
+    - lambda: |-
+        id(vbar3_effect_mode).set_level(x / 255);
+
+- platform: template
+  name: "VBar4 Effect Level"
+  icon: mdi:flower-pollen-outline
+  id: vbar4_effect_level_num
+  optimistic: true
+  min_value: 0
+  max_value: 255
+  step: 1
+#  mode: box
+  restore_value: true
+  set_action:
+    - lambda: |-
+        id(vbar4_effect_level).set_level(x / 255);
+- platform: template
+  name: "VBar4 Effect Mode"
+  icon: mdi:dots-hexagon
+  id: vbar4_effect_mode_num
+  optimistic: true
+  min_value: 0
+  max_value: 255
+  step: 1
+  mode: box
+  restore_value: true
+  set_action:
+    - lambda: |-
+        id(vbar4_effect_mode).set_level(x / 255);
+
+
+
+select:
+- platform: template
+  name: VBar1 Effect
+  optimistic: true
+  restore_value: true
+  icon: mdi:pan
+  id: vbar1_effect_select_dropdown
+  options:
+    - "None"
+    - "Color Macro"
+    - "Color Change"
+    - "Color Fade"
+    - "Sound Active"
+    - "Strobing"
+  set_action:
+    - lambda: |-
+          if (0 == x.compare(std::string{"None"})) {
```

Comment:
if(x.compare() == 0)

Otherwise, it doesn't match the coding style of the component.